src/api.ts

```diff
@@ -687,20 +679,20 @@ export function useProfileTodo(id: number) {
   });
 }
 
-export function useSector(
-  id: number | undefined,
-  options?: Parameters<typeof useData>[1],
-) {
-  return useData<any>(`/sectors?id=${id}`, {
-    ...(options ?? {}),
-    queryKey: [`/sectors`, { id }],
-  });
+export function useSector(id: number | undefined) {
+  return useData<(Success<"getSectors"> & { sectors: any[] }) | undefined>(
```

Comment:
FYI @jossi87 - `Sector.java` is also missing `getSectors()`, so it's dropped from the swagger definition too

Reply:
This is added now.

Reply:
Thanks!